error.go

```diff
@@ -55,6 +55,15 @@ func shouldRetry(err error, retryTimeout bool) bool {
 	if s == "ERR max number of clients reached" {
 		return true
 	}
+	if s == "Connection reset by peer" {
+		return true
+	}
+	if strings.HasPrefix(s, "ERR redis temporary failure") {
+		return true
+	}
+	if strings.HasPrefix(s, "FLUSHDB ") {
+		return true
+	}
```

Comment:
Please add a brief comment / reasoning why these errors must be retried.

Reply:
ok, I will add a comment / reasoning, thank you.